README.md

```diff
@@ -1,70 +1,266 @@
+
+#### Knife EC Backup/Restore
+If looking to change architecture, DB credentials, or even Pivotal then this is the best option.
+
+This can be down outside the entire Cloudformation process, but if wanting to do it inline see below.
+
+-   Required Items
+    -   [knife ec backup](https://github.com/chef/knife-ec-backup) in a tar file
+    -   Exisitng S3 Bucket with:
+        -   chef_ec_backups folder
+        -   Tar file from above in chef_ec_backups folder
+-   Required Params
+    -   ExistingBucketName (e.g.; chef-data-bucket)
+    -   BackupFilename (e.g.; backup_$date.tar)
+    -   DBPublic (You'll want to set this to `true` - If you wish to do external knife ec commands)
+    -   DBPublicCIDR (e.g.; 192.168.0.1/32, 172.33.0.1/16 - If you wish to do external knife ec commands)
+
+For External:
+
+If you want to sync down the S3 bucket utilized for the instance, there is a knife.rb in the root of the S3 bucket created at build time with all the config data to run the backup or restore commands.
+
+**Note:** be sure to check the required params above to make external backup/restore possible
+
+-   [knife ec](https://github.com/chef/knife-ec-backup) w/ the following items/info
+    -   Backup (See backup command below)
+    -   Info/Files Needed: (This comes included in the knife.rb synced to the S3 Bucket)
+        -   PostgreSQL Endpoint
+        -   DB User
+        -   DB Password
+        -   webui_priv.pem (found in /etc/opscode on existing server or S3 sync bucket under etc_opscode)
+-   Command to run
+    -   Without knife.rb in S3 Bucket
+        -   Backup: `knife ec backup /tmp/backup/ -s https://chef.domain.com --webui-key /tmp/webui_priv.pem --with-user-sql --sql-host some-db.rds.amazonaws.com --sql-user dbuser --sql-password sup3rs3cr3ts`
+        -   Restore: `knife ec restore /tmp/backup/ -s https://chef.domain.com --webui-key /tmp/webui_priv.pem --with-user-sql --sql-host some-db.rds.amazonaws.com --sql-user dbuser --sql-password sup3rs3cr3ts`
+    -   With knife.rb from S3 Bucket
+        -   Backup: `knife ec backup /tmp/backup/ -c /path/to/s3/knife.rb`
+        -   Restore: `knife ec restore /tmp/backup/ -c /path/to/s3/knife.rb`
+
+**Note:** You can do the backup just about anywhere, but doing the restore is more important. Doing it on a server in AWS in the same region is ideal as it will be able to upload much faster. Trying to upload from my local laptop has proven trying if not impossible due to the 5Mpbs max upload.
+
+## New Relic
+We utilize New Relic as our APM and System Monitor, this is setup only if conditions are met
+
+**If New Relic License Key Param is Filled Out**
+
+Installed via [newrelic.sh](newrelic.sh)
+
+-   What is Enabled:
+    -   New Relic APM (Gem Packaged with Chef)
+    -   NGINX Plugin via [MeetMe](https://github.com/MeetMe/newrelic-plugin-agent) Plugin Agent
```

Comment:
Specifically for Postgresql, https://newrelic.com/plugins/new-relic-platform-team/63

You have to have a special plugin for AWS services. There is not one for ElasticSearch right now.

Cloudwatch may be our best best for this, there is a plugin for cloudwatch but I have no idea what it entails to configure.